fenix/app/src/main/java/org/mozilla/fenix/debugsettings/redux/DebugDrawerStore.kt

```diff
@@ -0,0 +1,27 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.debugsettings.redux
```

Comment:
Could we call the package `store` instead of `redux`? I assume this is kinda how we do this https://searchfox.org/mozilla-mobile/search?q=Store.kt&path=. This is more to not mistaken this for ReduxJS.

Reply:
Sure thing